lunatrace/bsl/backend/src/tests/node-package-tree.test.ts

```diff
@@ -37,61 +78,72 @@ function testAllTreeTypes() {
   manifestFixtures.forEach((fixture) => {
     const fixturePath = path.resolve(__dirname, '../fixtures/manifests', fixture.name);
     describe(fixture.name, () => {
-      it(`should call hasura when calling snapshotPinnedDependecies`, async () => {
-        // build id doesnt matter, this never hits the DB
+      it(`should call hasura when calling snapshotPinnedDependencies`, async () => {
+        backup?.restore();
+        pgPromiseDb.none(`INSERT INTO builds (id, source_type) VALUES ('7f73f7ca-19f0-4e6a-92d5-e1deddd1319a', 'pr')`);
+        // build id doesn't matter, this never hits the DB
         await snapshotPinnedDependencies('7f73f7ca-19f0-4e6a-92d5-e1deddd1319a', fixturePath);
-        expect(hasura.InsertBuildDependencyRelationships).toBeCalled();
+        const manifests = await db.many<{ id: string }>(`SELECT id FROM manifest`);
+        expect(manifests.length).toBe(1);
+        expect(manifests[0].id).toBeTruthy();
       });
 
       it('collectPackageTreesFromDirectory should return one tree', async () => {
-        const trees = await collectPackageTreesFromDirectory(fixturePath);
+        const trees = await collectPackageGraphsFromDirectory(fixturePath);
         expect(trees.length).toEqual(1);
       });
 
-      it('should have no null ranges', async () => {
-        await snapshotPinnedDependencies('7f73f7ca-19f0-4e6a-92d5-e1deddd1319a', fixturePath);
-        const hasuraCallArgs = hasura.InsertBuildDependencyRelationships.mock.calls[0];
-        const objectsInserted = hasuraCallArgs[0].objects;
-
-        (objectsInserted as Array<Build_Dependency_Relationship_Insert_Input>).forEach((dep) => {
-          if (!dep.range) {
-            console.log('encountered dep with null range: ', util.inspect(dep, { depth: Infinity }));
-          }
-          expect(dep.range).toBeTruthy();
-        });
-      });
+      // TODO: Figure out if we want to make these tests work again or not.
+      // it('should have no null ranges', async () => {
+      //   await snapshotPinnedDependencies('7f73f7ca-19f0-4e6a-92d5-e1deddd1319a', fixturePath);
+      //   const hasuraCallArgs = hasura.InsertBuildDependencyRelationships.mock.calls[0];
+      //   const objectsInserted = hasuraCallArgs[0].objects;
+      //
+      //   (objectsInserted as Array<Build_Dependency_Relationship_Insert_Input>).forEach((dep) => {
+      //     if (!dep.range) {
+      //       console.log('encountered dep with null range: ', util.inspect(dep, { depth: Infinity }));
+      //     }
+      //     expect(dep.range).toBeTruthy();
+      //   });
+      // });
 
-      describe('root dependency counts', () => {
```

Comment:
Are you still checking these values are correct? Is the idea of the test alive in some other check or test somewhere?  Same for any other deleted test here. Are we still doing the tests somewhere in a different way but with the same intent?